aws/scripts/py/reports.py

```
@@ -51,33 +57,65 @@
+            generate_report_link(
+                bucket_name, region, build_id, reports_dir, file
+            )
+        )
 
     return links
 
 
 def copy_to_s3(source_dir, bucket_name, build_id, destination_dir):
-    subprocess.run(['aws', 's3', 'cp', f'{source_dir}/.',
-                   f's3://{bucket_name}/{build_id}/{destination_dir}', '--recursive'])
+    subprocess.run(
```

Ruff

`subprocess` call: check for execution of untrusted input

(S603)